server/settings.py

```diff
@@ -383,7 +383,7 @@ def env(variable, fallback_value=None):
 SUBSCRIPTION_LEVEL_SOLO = "solo"
 SUBSCRIPTION_LEVEL_TEAM = "team"
 SUBSCRIPTION_LEVEL_NETWORK = "network"
-SUBSCRIPTION_LEVEL = env("SUBSCRIPTION_LEVEL", SUBSCRIPTION_LEVEL_NETWORK)
+SUBSCRIPTION_LEVEL = env("SUBSCRIPTION_LEVEL", SUBSCRIPTION_LEVEL_GO)
```

Comment:
If we're ready to put this in a test instance we should revert this back. I'll ask systems engineers to put it on the go instance to play with it.